gradle.properties

```diff
@@ -1,4 +1,5 @@
 projectVersion=2.1.0-SNAPSHOT
+projectGroup=io.micronaut.problem
```

Comment:
Is this a change in 5.0.2? If you define `projectGroup` in `gradle.properties` then you don't need to specify the group in each module?

Reply:
Yes, that's correct. Since the pattern was introducing some repetition I made it a 1st class citizen.